requirements.txt

```diff
@@ -1,2 +1,3 @@
 flake8
+mock
```

Comment:
I know, but precise is quite old, and this file will be included in an existing package, so we don't want to add dependencies (python3)